lib/commands/start.js

```diff
@@ -37,10 +37,16 @@ class StartCommand extends Command {
 
             const start = () => {
                 return Promise.resolve(processInstance.start(process.cwd(), this.system.environment))
-                    .then(() => { instance.running(this.system.environment); });
+                    .then(() => {
+                        return processInstance.ensureStarted(process.cwd(), this.system.environment);
+                    })
+                    .then(() => {
+                        instance.running(this.system.environment);
+                    });
             };
 
             return this.ui.run(start, 'Starting Ghost', runOptions).then(() => {
+                // @TODO: shouldn't this happen before start?
```

Comment:
Needs a little discussion. I wondered why we enable the target process after we've started the instance.
If there is a good reason, we should explain it here.